config/jobs/kubernetes/sig-azure/sig-azure-config.yaml

```diff
@@ -0,0 +1,81 @@
+presets:
+- labels:
+    preset-azure-cred: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: REGISTRY
+    value: ritazh # update this after we have another account to use
+  - name: DOCKER_USERNAME
+    value: ritazh # update this after we have another account to use
+  - name: DOCKER_PASSWORD_FILE
+    value: /etc/docker-cred/dockercred
+  volumes:
+  - name: azure-cred
+    secret:
+      secretName: azure-cred
+  - name: azure-ssh
+    secret:
+      secretName: azure-ssh
+  - name: docker-cred
+    secret:
+      secretName: docker-cred
+  volumeMounts:
+  - name: azure-cred
+    mountPath: /etc/azure-cred
+    readOnly: true
+  - name: azure-ssh
+    mountPath: /etc/azure-ssh
+    readOnly: true
+  - name: docker-cred
+    mountPath: /etc/docker-cred
+    readOnly: true
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure
+    always_run: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: ritazh/kubekins-e2e:v20190126-731395d-dirty-master # update this after test-infra PRs are merged 
```

Comment:
which PRs?

Reply:
thanks, let's go ahead and push a new image for this first, xref #10958

Reply:
I pushed an image for `gcr.io/k8s-testimages/kubekins-e2e:v20190129-ee2861173-master`

Reply:
Thanks alot @BenTheElder! this PR is also pending  #11032

Reply:
ah sorry! will push again after we get that in :-)

Reply:
No, Thank you for being so responsive :)

Reply:
`gcr.io/k8s-testimages/kubekins-e2e:v20190129-fa5c18c02-master` is pushed now